webapp/api.py

```diff
@@ -42,3 +42,27 @@ def get_iot():
 
 def get_servers():
     return get_releases_by_vendor()
+
+
+def get_device_information_by_hardware_id(id):
+    model_info = (
+        get(f"certifiedmodels/?format=json&canonical_id={id}")
+        .json()
+        .get("objects")[0]
```

Comment:
Did you run `black` over this file? Could you?

Reply:
Yup, this line is 62 long